addons/skin.estuary/xml/DialogPVRChannelGuide.xml

```diff
@@ -75,14 +75,22 @@
 							<visible>!String.IsEmpty(ListItem.Plot) + Control.HasFocus(11)</visible>
 							<animation effect="fade" start="0" end="100" time="200">Focus</animation>
 							<control type="image">
-								<left>782</left>
+								<left>772</left>
 								<top>-100</top>
-								<width>900</width>
+								<width>1210</width>
 								<height>350</height>
 								<texture border="40">buttons/dialogbutton-nofo.png</texture>
 							</control>
+							<control type="image">
+								<left>805</left>
+								<top>-65</top>
+								<width>320</width>
+								<height>280</height>
+								<aspectratio align="center" aligny="center">keep</aspectratio>
+								<texture>$INFO[Listitem.Icon]</texture>
```

Comment:
If you change this line to `<texture fallback="DefaultTVShows.png">$INFO[Listitem.Icon]</texture>`you will get a default icon if neither an epg image nor a channel logo are available.